aws_gate/cli.py

```diff
@@ -29,14 +30,18 @@ def main():
     parser.add_argument('-v', '--verbose', help='increase output verbosity',
                         action='store_true')
     parser.add_argument('--version', action='version', version='%(prog)s {version}'.format(version=__version__))
-    subparsers = parser.add_subparsers(title='subcommands', dest='subcommand', metavar='{session}')
+    subparsers = parser.add_subparsers(title='subcommands', dest='subcommand', metavar='{session, ls}')
 
     # 'session' subcommand
     session_parser = subparsers.add_parser('session', help='Open new session on instance and connect to it')
     session_parser.add_argument('-p', '--profile', help='AWS profile to use', default=default_profile)
     session_parser.add_argument('-r', '--region', help='AWS region to use', default=default_region)
     session_parser.add_argument('instance_name', help='Instance we wish to open session to')
 
+    ls_parser = subparsers.add_parser('ls', help='List available instances and connect to it')
+    ls_parser.add_argument('-p', '--profile', help='AWS profile to use', default=default_profile)
+    ls_parser.add_argument('-r', '--region', help='AWS region to use', default=default_region)
```

Comment:
One of the use cases I had planned for list was to use it in shell completions. For that, I need to output the data in some machine-friendly format (e.g. display all the names/ips/domain names separated by tab or comma). However, this could be added once completion is actually available. Completion will be added after I finish config file support, which I have written, but I lack testing code (I am working on that now).